gov_uk_dashboards/axes/calculate_axis_range.py

```diff
@@ -0,0 +1,15 @@
+"""Function for calculating the axis range"""
+from math import floor, ceil
+import pandas as pd
+
+
+def calc_axis_range(df: pd.DataFrame, column: str) -> list:
```

Comment:
It would be valuable to bring over the [unit tests](https://github.com/communitiesuk/data_dashboards/blob/main/tests/unit/test_calculate_axis_range.py) to this repository too me thinks.

Reply:
Tests have been added and are excluded from the packaged up package too